index.html

```diff
@@ -0,0 +1,74 @@
+        <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+      </div>
+      <!-- SECTION ABOUT  -->
+      <div id="tribute-info">
+        <p>Here's a time line of Dr. Borlaug's life:</p>
+        <div>
+          <ul id="tribute-list">
+            <li><span>1914</span> - Born in Cresco, Iowa</li>
+            <li><span>1933</span> - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+            <li><span>1935</span> - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+            <li><span>1937</span> - Finishes university and takes a job in the US Forestry Service</li>
+            <li>
+              <span>1938</span> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.
+            </li>
+            <li>
+              <span>1941</span> - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied
+              science.
+            </li>
+            <li><span>1942</span> - Receives a Ph.D. in Genetics and Plant Pathology</li>
+            <li>
+              <span>1944</span> - Rejects a 100% salary increase from Dupont, leaves behind his pregnant wife, and flies to Mexico to head a new plant pathology program. Over the next 16 years, his team breeds 6,000 different strains of
+              disease resistent wheat - including different varieties for each major climate on Earth.
+            </li>
+            <li><span>1945</span> - Discovers a way to grown wheat twice each season, doubling wheat yields</li>
+            <li><span>1953</span> - crosses a short, sturdy dwarf breed of wheat with a high-yeidling American breed, creating a strain that responds well to fertilizer. It goes on to provide 95% of Mexico's wheat.</li>
+            <li><span>1962</span> - Visits Delhi and brings his high-yielding strains of wheat to the Indian subcontinent in time to help mitigate mass starvation due to a rapidly expanding population</li>
+            <li><span>1970</span> - receives the Nobel Peace Prize</li>
+            <li><span>1983</span> - helps seven African countries dramatically increase their maize and sorghum yields</li>
+            <li><span>1984</span> - becomes a distinguished professor at Texas A&M University</li>
+          </ul>
+        </div>
+      </div>
```

Comment:
Please use semantic element for the container 
```
<section>
...
</section>
```

You can change the div tag to section tag

Reply:
- [ ] Good job, mentioning the proper use of semantic tags here but It would be better if you can provide an explanation, a "why" for each requested change as this will help the person making the changes understand the reason & learn from their mistakes. ✔

Reply:
Please use semantic element for the container 
```
<section>
...
</section>
```

You can change the div tag to section tag

## Using the section tag under the main tag is important for several reasons:

1. Accessibility: Screen readers and other assistive technologies can use the section tag to help users navigate a web page. By using semantic tags like main and section, you can make your content more accessible to users with disabilities.

2. Search Engine Optimization (SEO): Search engines like Google use the HTML structure of a web page to understand the content and context of the page. By using semantic tags like main and section, you can help search engines better understand the structure and organization of your content, which can improve your search engine rankings.

3. Readability and maintainability: Using semantic tags like main and section can help make your code more readable and easier to maintain. By using tags that accurately describe the content and structure of your web page, it can be easier for other developers to understand and modify your code.

On the other hand, using the div tag does not provide any semantic meaning to the content and is typically used for general layout and styling purposes. While it is still possible to create a well-structured and accessible web page using only div tags, using semantic tags like main and section can provide additional benefits for accessibility, SEO, and code maintainability.